lib/validate.ts

```diff
@@ -68,10 +73,57 @@ export async function verifyCredential(credential: VerifiableCredential): Promis
       // Only check revocation status if VC has a 'credentialStatus' property
       checkStatus: hasRevocation ? checkStatus : undefined
     });
-
+    
     return result;
   } catch (err) {
     console.warn(err);
     throw new Error(CredentialErrorTypes.CouldNotBeVerified);
   }
 }
+
+function checkMalformed(credential: VerifiableCredential) {
+  let message = '';
+
+  // check credential for proof
+  if (!credential.proof){
+    message += 'This is not a Verifiable Credential (does not have a digital signature).'
+  }
+
+  if (message !== '') {
+    return {malformed: true, message: message};
+  }
+  return {malformed: false, message: message};
+
+}
+
+function createErrorMessage(credential: VerifiableCredential, message: string) {
+  return {
+    verified: false,
+    results: [
+      {
+        verified: false,
+        credential: credential,
+        error: {
+          details: {
+            cause: {
+              message: message,
+              name: 'Error',
+              stack: 'temp'
```

Comment:
Remove the `stack`, `code` and `url` properties if we're not using them. (If the receiving side on the front end is requiring them, change that.)